swap_meet/decor.py

```diff
@@ -1,2 +1,9 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self, category = "Decor", condition=0):
+        self.category = category
+        self.condition = condition
```

Comment:
same as above! Let's use inheritance and remove the spaces around the default parameters